### PR TITLE
Background Remover

### PR DESCRIPTION
**Pull request by DumzDev**

### CODE REVIEW

**Comment by Gen1x-ALT** [comment]
I ASKED + BASED + COOL + YOU ARE AWESOME + NOT SKILL ISSUE + W + GOOD EXTENSION

**Comment by RedMan13** [comment]
yeah it should be using something else that will be a bit more solid.........

**Comment by DumzDev** [comment]
> yeah it should be using something else that will be a bit more solid.........

theres a block to set the api key to whatever you want

**Comment by CodeFoxy-Github** [comment]
lgtm

File: static/extensions/dumzdev/removebg.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+                }
+            ],
+            menus: { },
+            color1: '#FFC536', // Main theme color
+            color2: '#FFD06B', // Lighter shade of #FFC536
+            color3: '#FFC536',
+            menuIconURI: 'https://files.catbox.moe/ezlx70.svg', // Menu icon
+            blockIconURI: 'https://files.catbox.moe/ezlx70.svg', // Block icon
+            textColor: '#000000', // Black text on the blocks
+            docsURI: 'https://www.remove.bg/api' // Button linking to Remove.bg API
```
Comment (SharkPool-SP) [inline static/extensions/dumzdev/removebg.js:47]: These should be before the blocks, not after.

Also the block and menu icons should be a data.uri not a direct url.

**Comment by SharkPool-SP** [comment]
You should also remove those comments from the colors, that's useless.

Also also maybe add a label that just warns that it won't work if the api key expires

File: static/extensions/dumzdev/removebg.js (hunk 1 above)
Comment (RedMan13) [reply static/extensions/dumzdev/removebg.js:47]: now the data uri thing is a hill I will die on over this cause needing to paste the raw image data into a JSON argument is terrible

Comment (DumzDev) [reply static/extensions/dumzdev/removebg.js]: done :)

**Pull request merged**